src/types/tuple.rs

```diff
@@ -458,4 +470,12 @@ mod test {
             );
         })
     }
+
+    #[test]
+    fn test_invalid_get_item() {
+        let gil = Python::acquire_gil();
+        let py = gil.python();
+        let ob = PyTuple::new(py, &[1, 2, 3]);
+        assert!(ob.get_item(100).is_err())
```

Comment:
👍 good idea to add a test, it could be nice to also check the error type (`IndexError`?) or message?